### PR TITLE
add deprecation warning about dev console

### PR DESCRIPTION
**Pull request by edunham**
This blog post has the following features:

- [x] Warns about obsolete information

If we have dedicated markdown formatting for admonition/warning boxes, oops! Point me at an example of a markdown post with a warning box and I'll mimic that instead.

### CODE REVIEW

File: _source/_posts/2017-09-25-all-new-developer-console.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ tags: [okta]
 type: awareness
 ---
 
+> :warning: This blog post was written in 2017. It is out of date.  Okta no longer provides a developer-centric admin console.  Instead, all administrators use the same admin console for configuring Okta.
```
Comment (edunham) [inline _source/_posts/2017-09-25-all-new-developer-console.md]: ``
```suggestion
> WARNING: This blog post was written in 2017. It is out of date.  Okta no longer provides a developer-centric admin console.  Instead, all administrators use the same admin console for configuring Okta. 
```

**Review by alisaduncan — APPROVED** [review]
🍠

**Comment by alisaduncan** [comment]
@edunham ok to merge this? If you want to use a standard format, there's updated front-matter you can use too. See this PR
https://github.com/oktadev/okta-blog/pull/1357

File: _source/_posts/2017-09-25-all-new-developer-console.md [hunk 2]
```diff
@@ -11,6 +11,10 @@ tags: [okta]
 type: awareness
 ---
 
+> WARNING: This blog post was written in 2017. It is out of date.  Okta no longer provides a developer-centric admin console.  Instead, all administrators use the same admin console for configuring Okta.
```
Comment (edunham) [inline _source/_posts/2017-09-25-all-new-developer-console.md]:
```suggestion
<div class="alert alert-info" role="alert">
  <p class="title">
    <a class="close" aria-label="Close" title="Close"
       onclick="this.parentElement.parentElement.style.display='none'">
      <span aria-hidden="true">&times;</span>
    </a>
    Heads up... this blog post is old!
  </p>
  <p>This blog post was written in 2017. It is out of date.  Okta no longer provides a developer-centric admin console.  Instead, all administrators use the same admin console for configuring Okta. 
  </p>
</div>
```

**Comment by edunham** [comment]
https://deploy-preview-1352--okta-blog.netlify.app/blog/2017/09/25/all-new-developer-console thanks for the tip on formatting, updated it and now we get a cute little warning. 
image

**Pull request merged**